From 2e4b6184c71d7e05f2550db3501119de554deb96 Mon Sep 17 00:00:00 2001
From: Jianing Ren <jianing.ren@rock-chips.com>
Date: Fri, 18 Oct 2019 18:19:56 +0800
Subject: [PATCH] HACK: arm64: dts: rk3399-android: fixing the adb function
 cannot be used

This is a HACK patch setting dwc3 dr-mode from otg to peripheral.

Change-Id: I4a6ba065e0369315e95f6ad251b58cc16b1c4e81
Signed-off-by: Jianing Ren <jianing.ren@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-android.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi
index 1d3f7722ffa2..e79a2712d8b4 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi
@@ -354,7 +354,7 @@
 };
 
 &usbdrd_dwc3_0 {
-	dr_mode = "otg";
+	dr_mode = "peripheral";
 };
 
 &iep {
-- 
2.35.3

